rust/src/api/lbtc_ln.rs

```diff
@@ -351,6 +351,7 @@ pub struct LbtcLnV2Swap {
     pub blinding_key: String,
     pub electrum_url: String,
     pub boltz_url: String,
+    pub referral_id: String,
```

Comment:
Maybe better to make this also Option<String> that way in dart we get String?

Better than an empty string.

Reply:
i missed, this one.

Fixed